spring-cloud-gcp-samples/spring-cloud-gcp-vision-api-sample/README.adoc

```diff
@@ -4,7 +4,7 @@ This code sample shows how the Spring Cloud GCP integrations can simplify the us
 Cloud Platform Java Client Libraries.
 
 It uses the https://cloud.google.com/vision/[Google Cloud Vision API] to extract information about a picture.
-We take advantage of the link:/spring-cloud-gcp-starters/spring-cloud-gcp-starter-core[Core starter] to retrieve
+We take advantage of the link:/spring-cloud-gcp-starters/spring-cloud-gcp-starter[Core GCP Support starter] to retrieve
```

Comment:
Looks like the convention is do not mention `core` word at all...

Let me know if this is OK, or like you have left in the previous one - "using GCP Starter Core properties"

Reply:
It's fine by me. I'll make the change.